plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConfig.java

```diff
@@ -75,7 +75,8 @@ public void testDefaults()
                 .setQueryPartitionFilterRequired(false)
                 .setDeletionVectorsEnabled(false)
                 .setDeltaLogFileSystemCacheDisabled(false)
-                .setMetadataParallelism(8));
+                .setMetadataParallelism(8)
+                .setVacuumTransactionLoggingEnabled(false));
```

Comment:
Move this line under `setVacuumMinRetention`. Same for other changes in this class.